error_handler.py

```diff
@@ -0,0 +1,22 @@
+import discord
+import asyncio
+
+class Error_handler():
+    def __init__(self):
+        self.error_messages = {}
```

Comment:
should be a private variable -> use OOP encapsulation

Reply:
Privatised a94ae30